src/test/resources/feature/3-destinatario/personaGiuridica/deleghePG/008_115_ricercaDelegheConGruppoPG.feature

```diff
@@ -9,6 +9,17 @@ Feature:La persona giuridica fa una ricerca per gruppo delle deleghe
   Scenario: PN-9167 - La persona giuridica fa una ricerca per gruppo delle deleghe
     Given PG - Si effettua la login tramite token exchange come "delegante", e viene visualizzata la dashboard
     When Nella pagina Piattaforma Notifiche persona giuridica click sul bottone Deleghe
+    And Nella pagina Deleghe si clicca su Delegati dall impresa
+    And Si visualizza correttamente la pagina Deleghe sezione Deleghe dell impresa
+    And Creo in background una delega per persona giuridica
+      | accessoCome    | delegante         |
+      | fiscalCode     | LELPTR04A01C352E  |
+      | companyName    | Le Epistolae srl  |
+      | displayName    | Le Epistolae srl  |
+      | person         | false             |
+    And Nella sezione Delegati dall impresa si visualizza correttamente una delega in stato di attesa di conferma "Le Epistolae srl"
+    And Si revoca delega come delegante con api
+    And Nella pagina Deleghe si clicca su Deleghe a carico dell impresa
```

Comment:
This `feature step` too doesn't check if there is the `table` element exist

Reply:
done